exercises/02-bad-flask-app-intro.md

```diff
@@ -2,14 +2,14 @@
 
 ## Learning Goals
 
-- Understand general good steps to do at the beginning of developing
+- Understanding of general good steps to do at the beginning of developing
 test cases.
 
 ## Introduction
 
-This is the UI portion of the training. The overall goal for the rest of
-the exercices is to fill the form successfully. The exercises will not necessarily get
-more difficult as they go on, but will tackle completely different issues.
+This is the UI portion of the training. The final goal for the rest of
+the exercices is to fill out the form successfully. The exercises will not necessarily get
+more difficult, but will tackle completely different issues.
```

Comment:
"different issues" is a bit vague, are we meaning "will tackle variety of separate topics", or something like this?

Reply:
Yes, it means the exercises will handle different topics.